getting-started/data-setup.rst

```diff
@@ -46,149 +44,91 @@ It may simply be that you're working in a brand new shell.
+Instead, you access data through and instance of the **Butler** class.
 This gives you flexibility to work with data from different observatories without significantly changing your workflow.
 
 The Butler manages data in **repositories.**
 Butler repositories can be remote (the data is on a server, across a network) or local (the data in on a local filesystem).
 In this tutorial you'll create and use a local Butler repository, which is a simple directory.
 
-Go ahead and create the local Butler repository as a directory called :file:`DATA`:
-
-.. code-block:: bash
-
-   mkdir DATA
-
-Then add a :file:`_mapper` file to the repository:
-
-.. code-block:: bash
-
-   echo "lsst.obs.hsc.HscMapper" > DATA/_mapper
-
-The Butler uses the **mapper** to find and organize data in a format specific to each camera.
-Here you're using the ``lsst.obs.hsc.HscMapper`` mapper because you're processing HSC data in this repository.
-
-This is what your current working directory should look like right now:
-
-.. code-block:: text
-
-   testdata_ci_hsc/
-   DATA/
-
-Ingesting raw data into the Butler repository
-=============================================
-
-Next, populate the repository with data from `testdata_ci_hsc`_.
-The Pipelines' :command:`ingestImages.py` command (called a **command-line task**) links raw images into a Butler repository, allowing the mapper to organize the data.
-Run:
-
-.. code-block:: bash
-
-   ingestImages.py DATA $TESTDATA_CI_HSC_DIR/raw/*.fits --mode=link
-
-.. tip::
-
-   Notice that the first argument to most command-line tasks is the Butler repository.
-   In this case it's the :file:`DATA` directory.
-
-.. tip::
-
-   You can learn about the arguments for command-line tasks with the ``-h`` flag.
-   For example:
-
-   .. code-block:: bash
+The `gen3_rc2_subset`_ git repository has a Butler repository contained within it.
+To construct a Butler that can manage data in that repository, from a python prompt say:
 
-      ingestImages.py -h
+.. code-block:: python
 
-Install transmission curves
-===========================
+   from lsst.daf.butler import Butler
+   import os
+   butler = Butler(os.environ['GEN3_DC2_SUBSET_DIR'] + '/SMALL_HSC')
 
-Run this command to install transmission curves corresponding to the raw data:
+Now you can explore the repository using the registry atribute of the Butler you created.  E.g.:
 
-.. code-block:: bash
-
-   installTransmissionCurves.py DATA
-
-Transmission calibrations, like this, are currently a special feature for HSC data `implemented in the obs_subaru package <https://github.com/lsst/obs_subaru/tree/master/hsc/transmission>`_.
+.. code-block:: python
 
-Ingesting calibrations into the Butler repository
-=================================================
+   registry = butler.registry
+   for col in registry.queryCollections():
+       print(col)
+   for ref in registry.queryDatasets('raw', collections='', instrument='HSC'):
+       print(ref.full)
```

Comment:
This `.full` seems confusing without explanation. Also, does it actually work? Isn't it meant to be `ref.dataId.full` ?